editoast/src/views/test_app.rs

```diff
@@ -158,6 +173,22 @@ where
             .expect("no DbConnectionPoolV2 setup")
             .clone()
     }
+
+    pub fn fetch(&self, req: Request) -> TestResponse {
+        let uri = format!("{} {}", req.method(), req.uri());
+        let span = tracing::debug_span!("Request", uri);
```

Comment:
Why not putting them as metadata?

```suggestion
        let span = tracing::debug_span!(method = req.method(), uri = req.uri(), "Request", uri);
```

This way, you could configure `RUST_LOG=[{method=POST}]` to filter only the POST calls. See https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html.